images/nginx/Makefile

```diff
@@ -13,24 +13,26 @@
 # limitations under the License.
 
 .DEFAULT_GOAL:=build
-
+DATE := $(shell date +%Y%m%d)
+GIT_COMMIT := $(shell git log -1 --pretty=format:%h)
 # set default shell
-SHELL=/bin/bash -o pipefail -o errexit
+SHELL = /bin/bash -o pipefail -o errexit
 
-DIR:=$(strip $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+DIR := $(strip $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
 INIT_BUILDX=$(DIR)/../../hack/init-buildx.sh
 
 # 0.0.0 shouldn't clobber any released builds
-TAG ?= 0.0
-REGISTRY ?= gcr.io/k8s-staging-ingress-nginx
+TAG ?= $(DATE)-$(GIT_COMMIT)
```

Comment:
Is there an ability to use tags there ? With this approach we probably have to tag release image manually, am i right ?

Reply:
Not really, if `TAG` environment variable isn't set, the tag will be generated according to `TAG ?= $(DATE)-$(GIT_COMMIT)` scheme. e.g. https://hub.docker.com/layers/infoblox/nginx-fips/20200908-bcd33a8d2/images/sha256-f16dee0ab2fdfabf0f70163f071682d350970906d52fc5b730c0a413f3cdc925?context=repo

Reply:
Cool, thanks for clarification

Reply:
No concerns from my side